connect/README.md

```diff
@@ -0,0 +1,259 @@
+<h1 align="center">⚡️ hyper connect workshop ⚡️</h1>
+<p align="center">In this workshop, we will dive deeper into hyper-connect and learn some more Data service methods</p>
+
+---
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [What is hyper-connect?](#what-is-hyper-connect?)
+- [Setup](#setup)
+- [Update a character](#update-a-character)
+- [Remove a character](#remove-a-character)
+- [List characters](#list-characters)
+- [Summary](#summary)
+
+---
+
+## Introduction 
+
+👋🏻 Greetings, Thank you for taking the time to learn hyper with us, in this workshop, we
+will dive a little deeper into hyper-connect the deno/nodejs hyper client. But first, if you
+have not taken the [Hello World](../hello-world) workshop, please consider to take that workshop
+first, we will be building on the concepts we learned in previous workshops.
+
+> NOTE: Please be sure to take the [Hello World](../hello-world) workshop before taking this workshop.
+
+As we started in the hello-world workshop, we will be building out a "Mario Wiki API", and through these
+workshops, we will cover all of the hyper core services. The best way to learn is by doing, so in order 
+to get the most out of these workshops, take the time to actually do the work, not just watch or copy
+and paste.
+
```

Comment:
We'll continue building out the "Mario Wiki API" we started in the [Hello World](../hello-world) workshop.  We will cover all of the core hyper services. The best way to learn is by doing. In order 
to get the most out of these workshops, take the time to actually do the work, not just watch or copy
and paste.